src/os/ui/draw/drawpicker.js

```diff
@@ -441,3 +447,17 @@ os.ui.draw.DrawPickerCtrl.prototype.onCountryChosen = function(garbage, feature)
     this.scope['callback'](geometry);
   }
 };
+
+
+/**
+ * Handler for water chosen.
+ *
+ * @param {ol.Feature} garbage The chosen water.
```

Comment:
Odd variable name...

Reply:
rofl - I think the original writer of onCountryChosen() was frustrated by the difference between the "name, id" ONLY feature and the fully-loaded-from-the-FeatureLayer feature... 
Anyway, this is gone now and I updated onCountryChosen() to work generically instead.